packages/base-contract/src/index.ts

```diff
@@ -44,6 +44,7 @@ export class BaseContract {
         const constructorAbiIfExists = _.find(
             abi,
             (abiDefinition: AbiDefinition) => abiDefinition.type === AbiType.Constructor,
+            // tslint:disable-next-line:no-unnecessary-type-assertion
```

Comment:
Why were these removable now?

Reply:
Not sure what you mean.

Reply:
Why were you able to remove these unnecessary type assertions, whereas before we disabled the tslint rule?

Reply:
nvm!